# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 202c4f5c642b
# Full Hash: 202c4f5c642b8ec35e849c2144f935342d193fd7
# Author: Vincent Hilla <vhilla@mozilla.com>
# Date: 2023-12-05 21:29:44
# Regressor Bug: 1589554
# File Overlap Count: 1
# Description:
#   Bug 1589554 - Part 3: Screen Wake Lock permission integration. r=webidl,dom-core,smaug,pbz,edgar
#   
#   Depends on D189509
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D189510
# ==============================================================================

diff -r 82e03a404c2f -r 202c4f5c642b modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue Dec 05 15:37:05 2023 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue Dec 05 15:37:06 2023 +0000
@@ -12818,6 +12818,11 @@
   value: 1
   mirror: always
 
+- name: permissions.default.screen-wake-lock
+  type: RelaxedAtomicUint32
+  value: 1
+  mirror: always
+
 - name: permissions.isolateBy.userContext
   type: RelaxedAtomicBool
   value: false
